### DISCUSSION TITLE
Feature request: Add option to place the sidebar on the left or right side of the screen.

### DISCUSSION THREAD

**Discussion by xdubx** [SchwarzIT/onyx#2301] [Ideas]
title 👍 

link to ticket https://github.com/SchwarzIT/onyx/issues/2262

**Comment by mj-hof** [comment]
Hi @xdubx. Thank you for your request. It's planned to implement the possibility to decide on which side the sidebar docks. See the screenshot of the concept below and the implementation ticket #2726 

<img width="877" alt="Bildschirmfoto 2025-03-04 um 15 57 25" src="https://github.com/user-attachments/assets/391759e4-f362-498a-9914-acf4ac73cedf" />
